[Workflow] Updates for Go SDK (#3895)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

{{% /codetab %}}

<!--Java-->
{{% codetab %}}

> Manage your workflow within your code. [In the workflow example from the Java SDK](https://github.com/dapr/java-sdk/blob/master/examples/src/main/java/io/dapr/examples/workflows/DemoWorkflowClient.java), the workflow is registered in the code using the following APIs:

Reviewer: Link is broken here
```suggestion
Manage your workflow within your code. [In the workflow example from the Java SDK](https://github.com/dapr/java-sdk/blob/master/examples/src/main/java/io/dapr/examples/workflows/), the workflow is registered in the code using the following APIs:
```

{{% /codetab %}}

<!--Go-->
{{% codetab %}}

> Manage your workflow within your code. [In the workflow example from the Go SDK](todo), the workflow is registered in the code using the following APIs:

Reviewer: Missing link here in the (todo)

> Manage your workflow within your code. [In the workflow example from the Go SDK](todo), the workflow is registered in the code using the following APIs:

Reviewer 2: ```suggestion
Manage your workflow within your code. [In the workflow example from the Go SDK](https://github.com/dapr/go-sdk/tree/main/examples/workflow), the workflow is registered in the code using the following APIs:
```

  - [JavaScript example](https://github.com/dapr/js-sdk/tree/main/examples/workflow)
  - [.NET example](https://github.com/dapr/dotnet-sdk/tree/master/examples/Workflow)
  - [Java example](https://github.com/dapr/java-sdk/tree/master/examples/src/main/java/io/dapr/examples/workflows)

>   - [Go example](todo)

Reviewer: Another todo to resolve

>   - [Go example](todo)

Reviewer 2: ```suggestion
  - [Go example](https://github.com/dapr/go-sdk/tree/main/examples/workflow)
```
